imports/plugins/included/product-detail-simple/client/containers/productDetailContainer.js

```diff
@@ -112,14 +115,15 @@ class ProductDetailContainer extends Component {
             [oppositeDirection]: "auto",
             [direction]: 0
           }, 600)
-          .delay(4000)
+          .delay(2000)
           .animate({
             [oppositeDirection]: "auto",
             [direction]: -alertWidth
           }, {
             duration: 600,
             complete() {
```

Comment:
Is there a way to re-enable add to cart at the start rather than the end of this animation?